custom-footer.js

```diff
@@ -126,6 +160,32 @@ $(function () {
         // Rewrite heading for incidents list to make it clear it includes today
         document.querySelector('.incidents-list > h2:first-child').textContent = 'Recent incidents'
 
+        // make components list an actual semantic list
+        // adds aria-describedby to the component name container to announce the status
+        // adds visually hiddent 'status' and 'comppnent' texts
+        const $componentContainer = document.querySelector('.components-container')
+        swapElForHTML(document.querySelector('.components-container'), `<ul class="${$componentContainer.className}">${$componentContainer.innerHTML}</ul>`)
+        const $newComponentContainer = document.querySelector('.components-container')
+        const $components = $newComponentContainer.querySelectorAll('.component-container')
+        $components.forEach($el => {
+          const classes = $el.className
+          const $componentName = $el.querySelector('.name')
+          const $componentStatus = $el.querySelector('.component-status')
+          const componentStatusClasses = $el.querySelector('.component-inner-container').className
+          const $listElement = `
+          <li class="${classes} ${componentStatusClasses}">
+            <span class="${$componentName.className}">
+              <span class="govuk-visually-hidden">component name: </span>
```

Comment:
I'm not sure we need this hidden prefix. The visual interface doesn't give that information and you get some of it from knowing the status is a status. It's totally my preference but, when using voiceover to go through the components, the component name prefix felt unnecessary pretty quickly.

Reply:
removed